tree/20_cfe_basics/README.md

```diff
@@ -4,3 +4,10 @@
 
 This directory contains librairies with utility bodies that can be reused. Most notably, it contains the cfengine standard librairy.
 
+The `cfengine` folder contains the CFEngine stdlib from 3.10.
+
+The changes made are:
+
+* Removing some files we do not use: testing.cf, autorun.cf, stdlib.cf, examples.cf, cfe_internal_hub.cf, cfengine_enterprise_hub_ha.cf
```

Comment:
if the goal is to use integration with existing installations, we should keep autorun, cfe_internal_hub and cfengine_enterprise_hub_ha